src/tags/object/TimeSeries.js

```diff
@@ -28,6 +28,8 @@ import PersistentStateMixin from "../../mixins/PersistentState";
 import "./TimeSeries/Channel";
 import { AnnotationMixin } from "../../mixins/AnnotationMixin";
 
+const moment = require("moment");
+
```

Comment:
Moment is not explicitly defined as a dependency of this project, and I don't think it is needed here.
```suggestion
```

Reply:
Agreed, will be removed by refactoring formatDuration.